From 927a9d5ac4a287089be3589ea008c3eff66f1d28 Mon Sep 17 00:00:00 2001
From: Mykola Lysenko <Mykola.Lysenko@amd.com>
Date: Wed, 9 Dec 2015 20:30:37 +0800
Subject: [PATCH 1152/1565] drm/amdgpu: add semaphore to connector for mst
 syncronization

When MST branch got disconnected we need to wait to delete
MST connector, and make sure that reset mode occured on that connector

Change-Id: I9d90d4fe722b34989456b41db2effe37e9d8a940
Signed-off-by: Mykola Lysenko <Mykola.Lysenko@amd.com>
Reviewed-by: Jordan Lazare <Jordan.Lazare@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h b/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
index 9a13b9f..6ebb150 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
@@ -556,6 +556,7 @@ struct amdgpu_connector {
 	struct amdgpu_connector *mst_port;
 	bool is_mst_connector;
 	struct amdgpu_encoder *mst_encoder;
+	struct semaphore mst_sem;
 };
 
 /* TODO: start to use this struct and remove same field from base one */
-- 
1.9.1

